integrations/integration-vip-config.php

```diff
@@ -159,26 +152,19 @@ public function get_site_config() {
 	 * @return null|string|array Returns `null` if key is not found, `string` if key is "status" and `array` if key is "config".
```

Comment:
This seems confusing and not entirely accurate? There's no special handling for `config` vs `status`

Reply:
No special handling, but in the configmap the `config` is an array and `status` is a string. So the return type does vary based on that.